kmip/services/server/crypto/engine.py

```diff
@@ -459,7 +492,10 @@ def _encrypt_symmetric(
                 if iv_nonce is None:
                     iv_nonce = os.urandom(algorithm.block_size // 8)
                     return_iv_nonce = True
-                mode = mode(iv_nonce)
+                if is_gcm_mode:
+                    mode = mode(iv_nonce, None, auth_tag_length)
```

Comment:
Let's use `tag=None` and `min_tag_length=auth_tag_length` here, just to be explicit.